server/lib/routes/post-metrics-errors.js

```diff
@@ -91,31 +91,21 @@ const QUERY_SCHEMA = {
 /**
  * Attaches extra tags to sentry data
  *
- * @param {String} data - stringified Sentry data object
+ * @param {String} sentryData - stringified Sentry data object
```

Comment:
Since this works directly with the data object, `stringified` no longer seems applicable, and the type should be `Object`.